pom.xml

```
@@ -853,7 +853,7 @@
                                         <!-- JUnit 5 is the preferred test framework -->
                                         <exclude>org.testng:testng</exclude>
                                         <!-- Cat-X Deps -->
-                                        <exclude>org.json:json:*:*:compile</exclude>
+                                        <exclude>org.json:json:[,20240303]:compile</exclude>
```

Reviewing the [Maven Version Ranges](https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html), it looks like this should use parentheses to indicate `x < 20240303`:
```suggestion
                                        <exclude>org.json:json:(,20240303):compile</exclude>
```